src/form/state/FormFieldDeclaration.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface FormFieldDeclaration {
+  fieldType?: 'checkbox' | 'checkboxgroup' | 'date' | 'datetime' | 'input' | 'radiogroup' | 'select' | 'time';
```

Comment:
Did you not include a single `radiobutton` by design, as radio options should only be used if there's 2 or more?

Reply:
Indeed a single checkbox is allowed, whereas a single radio button would not be useful.

Reply:
I do think `textarea` is probably missing here.